jib-maven-plugin/gradle.properties

```diff
@@ -1 +1 @@
-version = 2.2.1-SNAPSHOT
+version = 2.3.1
```

Comment:
Shouldn't this have been 2.3.1-SNAPSHOT?

Reply:
yeah you're right, good thing for code reviews